bucket/archi.json

```diff
@@ -5,7 +5,7 @@
     "license": "MIT",
     "architecture": {
         "64bit": {
-            "url": "https://www.archimatetool.com/downloads/archi.php?/5.1.0/Archi-Win64-5.1.0.zip",
+            "url": "https://www.archimatetool.com/downloads/archi5.php?/5.1.0/Archi-Win64-5.1.0.zip",
```

Comment:
Hello !

A little hyphen is missin here. The URL should be : 

`https://www.archimatetool.com/downloads/archi-5.php?/5.1.0/Archi-Win64-5.1.0.zip`

Reply:
thanks, the URL seems to changed again.